tests/run-make/short-ice/rmake.rs

```diff
@@ -0,0 +1,41 @@
+// Backtraces in internal compiler errors used to be unbearably long, spanning
+// multiple hundreds of lines. A fix was pushed in #108938, and this test gathers
+// varied metrics on level 1 and full-level backtraces to check that the output
+// was shortened down to an appropriate length.
+// See https://github.com/rust-lang/rust/issues/107910
+
+use run_make_support::rustc;
+use std::env;
+
+fn main() {
+    env::set_var("RUST_BACKTRACE", "1");
+    let mut rust_test_1 = rustc().input("src/lib.rs").arg("-Ztreat-err-as-bug=1").command_output();
```

Comment:
Suggestion: this should use `run_fail()`, cc https://github.com/rust-lang/rust/pull/125752.